src/test/java/HereApi/XMLParserTest.java

```diff
@@ -0,0 +1,28 @@
+package HereApi;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class XMLParserTest {
```

Comment:
Ich glaube, dies ist nur der Dummy-Test bzw. das Test-Beispiel. Das würde ich in dieser Form nur ungern in den Master übernehmen wollen.